tests/Cargo.toml

```diff
@@ -47,6 +47,7 @@ futures = "0.3"
 hyper = { workspace = true, features = ["server"] }
 insta = { workspace = true }
 itertools = { workspace = true }
+portpicker = "0.1"
```

Comment:
If we use stuff from `fuel-core-bin`, then we don't need to pick a port=)

Reply:
Really? I think running two tests like that in parallel will break if we do not randomize ports.